aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -3168,6 +3168,18 @@ swf:
   tags:
     product_categories: {}
   version: swf-2012-01-25
+synthetics:
+  api_ref: AmazonSynthetics/latest/APIReference/Welcome.html
+  blurb: ''
+  expanded:
+    long: Amazon CloudWatch Synthetics
+    short: CloudWatch Synthetics
+  long: '&CWlong; Synthetics'
```

Comment:
Missing `guide` field.

Reply:
I can't find a guide for CW Synthetics, only this page in the CW guide: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html

Can you find something better?